### PR TITLE
docs: Update docs for OVMS and model formats

### PR DESCRIPTION
**Pull request by aluu317**
Signed-off-by: Angel Luu <angel.luu@us.ibm.com>

"Fixes #issue-number" or "Add description of the problem this PR solves"
Update docs to reflect the new support for Open VINO model server and its model type OpenVINO IR

## Proposed Changes 

-
-
-

### CODE REVIEW

File: docs/modelserving/mms/modelmesh/overview.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ System-wide configuration parameters can be set by [creating a ConfigMap](https:
 
 - [triton-inference-server](https://github.com/triton-inference-server/server) - NVIDIA's Triton Inference Server
 - [seldon-mlserver](https://github.com/SeldonIO/MLServer) - Python-based inference server
+- [openVINO-model-server](https://github.com/openvinotoolkit/model_server) - OpenVINO Model Server
```
Comment (yuzisun) [inline docs/modelserving/mms/modelmesh/overview.md:44]: Is this runtime installed part of ModelMesh right?

Comment (aluu317) [reply docs/modelserving/mms/modelmesh/overview.md:44]: yes, it was merged  as part of this PR https://github.com/kserve/modelmesh-serving/pull/141

Comment (aluu317) [reply docs/modelserving/mms/modelmesh/overview.md:44]: @yuzisun Is there any other concern/question, we'd like to get this merged soon

**Pull request merged**